tests/translate/translate_fillz.py

```diff
@@ -53,5 +53,5 @@ def compute(self, inputs):
         for varname, data in inputs["tracers"].items():
             index = utils.tracer_variables.index(varname)
             tracers[:, :, index] = np.squeeze(data[self.grid.slice_dict(ds)])
-        out = {"q2tracers": tracers}
-        return out
+        self.ignore_near_zero_errors = {"q2tracers": True}
```

Comment:
We usually have this in the init method by convention, but it does work here as well!

Reply:
Removed, I will save this for the GPU tolerance PR so it can be set per backend.